[addlyrics] lyrics to Kashmir

RitikaD16 opened the pull request:
Inspired during a drive through a desert area in Morocco...none of the group members had visited Kashmir.

dianagold commented:
@vkhademi yay! you have successfully created the pull request and merged your branch into `main`. Now your branch `vahab` can be safely deleted. It is good practice to delete a branch **after** you merge it - you can always recreate one with the same name if you want to, and that ensures a healthy dose of pruning. You should see a button `Delete branch` in this page!

RitikaD16 committed:
[addlyrics] lyrics to Kashmir

Inspired during a drive through a desert area in Morocco...none of the group members had visited Kashmir.
